tutorials/cp-mobile-bas-setup/cp-mobile-bas-setup.md

```diff
@@ -96,7 +95,7 @@ Your dev space will be created and the status will change to running.
 
     !![Space list](img_3_6.png)
 
-    > Upon successful setup, you will see a toast message at the bottom right corner of your screen, and the bottom status bar will highlight CF connection details.
+    > Upon successful set up, you will see a toast message at the bottom right corner of your screen, and the bottom status bar will highlight CF connection details.
```

Comment:
Here you are referring to the setup as such and not the task. Please undo this change.

Reply:
done

Reply:
oh wait 
![image](https://user-images.githubusercontent.com/31536700/107780617-35580a80-6d47-11eb-9376-3f6a602a2c4b.png)

Reply:
finally done